src/php/ext/grpc/call.c

```diff
@@ -537,7 +558,9 @@ PHP_METHOD(Call, startBatch) {
     }
     if (ops[i].op == GRPC_OP_RECV_MESSAGE) {
       grpc_byte_buffer_destroy(message);
+      #if PHP_MAJOR_VERSION < 7
```

Comment:
Why only free for php 5? Oh because it has already been `zval_ptr_dtor` above?

Reply:
yes